src/main/java/org/broadinstitute/barclay/help/HelpDoclet.java

```diff
@@ -127,26 +146,43 @@ else if (!settingsDir.isDirectory())
      * methods defined here to allow default handling of builtin options.
      *
      * @param options Options to parse.
+     * @return True if {@code options} was parsed, False otherwise.
      */
-    protected void parseOption(final String[] options) {
-        if (options[0].equals(SETTINGS_DIR_OPTION))
+    protected boolean parseOption(final String[] options) {
+
+        boolean hasParsedOption = false;
```

Comment:
I've always thought that for the most part it's a little less complex to have a single return from a function.